xdsl/dialects/experimental/stencil.py

```diff
@@ -103,9 +102,7 @@ def min(a: IndexAttr, b: IndexAttr | None) -> IndexAttr:
     def max(a: IndexAttr, b: IndexAttr | None) -> IndexAttr:
         if b is None:
             return a
-        return IndexAttr.get(
-            *(max(ae.data, be.data) for ae, be in zip(a.array.data, b.array.data))
-        )
+        return IndexAttr.get(*(max(ae, be) for ae, be in zip(a, b)))
```

Comment:
```suggestion
        return IndexAttr.get(*map(max, a, b))
```
Supernit, I think this should work

Reply:
Lovely, indeed, thanks!